hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -304,6 +306,11 @@ private ContainerCommandResponseProto dispatchRequest(
         // Once container is marked unhealthy, all the subsequent write
         // transactions will fail with UNHEALTHY_CONTAINER exception.
 
+        if (container == null) {
+          throw new RuntimeException(
```

Comment:
May be better to throw a more specific exception (`IllegalState` or `NullPointer`)?